understanding/20/use-of-color.html

```diff
@@ -154,21 +154,34 @@ <h2>Benefits of Use of Color</h2>
    <section id="examples">
       <h2>Examples of Use of Color</h2>
       
-      <dl>
-         <dt>A form that uses color and text to indicate required fields</dt>
-         <dd>A form contains both required and optional fields. Instructions at the top of the
+      <h3>A form that uses color and text to indicate required fields</h3>
+         <p>A form contains both required and optional fields. Instructions at the top of the
                form explain that required fields are labeled with red text and also with an icon.
                Users who cannot perceive the difference between the optional field labels and the
                red labels for the required fields will still be able to see the icon next to the
-               red labels.</dd>
-         <dt>An examination</dt>
-         <dd>Students view an SVG image of a chemical compound and identify the chemical elements
+               red labels.</p>
+     
+         <h3>An examination</h3>
+         <p>Students view an SVG image of a chemical compound and identify the chemical elements
                present based <strong>both</strong> on the colors used, as well as numbers next to each
                element. A legend shows the color and number for each type of element. Sighted users who
                cannot perceive all the color differences can still understand the image by relying on
-               the numbers.</dd>
-      </dl>
-      
+               the numbers.</p>
+     
+         <h3>Focus on links</h3>
+         <p>A block of text has a text link in blue that has a contrast of 3:1 to the surrounding text. When this link receives keyboard focus, it turns into a darker blue. The contrast difference between the link color in the focused and the unfocused state is only 2.0:1. In addition, after receiving focus the contrast difference between the link and the surrounding text is now also well below 3:1. To show the keyboard focus clearly, the link is in addition underlined, so the change of color is not the only indicator of keyboard focus. Without an additional focus indicator the example would fail the Success Criterion 1.4.1 Use of Color.</p>
```

Comment:
just some wording suggestions

```suggestion
         <p>A block of text contains a blue hyperlink that has a contrast of 3:1 to the surrounding text. When receiving keyboard focus, the link's text color changes to a darker blue. The contrast difference between the link color in the focused and the unfocused state is only 2:1. In addition, when focused the contrast difference between the link's text color and the surrounding text falls well below a 3:1 ratio. However, to further emphasize the focused state the link is provided an addition underlined. Due to the additional underline, color is not used as the only visual indicator of the focused hyperlink, so it passes Success Criterion 1.4.1 Use of Color. Without the underline, the hyperlink would have failed this Success Criterion.</p>
```

Reply:
Suggestion to clarify the link's visual design:

Current: "A block of text contains a blue hyperlink that has a contrast of 3:1 to the surrounding text."
Proposed: "A block of text contains a non-underlined blue hyperlink that has a contrast of 3:1 to the surrounding text."

Typo: "provided an _addition underlined_".

Reply:
thank you @fstrr.  incorporated into my suggested update